src/main/java/io/vertx/ext/apex/core/impl/Utils.java

```diff
@@ -115,7 +116,7 @@ public static String readResourceToString(String resource) {
   }
 
   public static DateFormat createISODateTimeFormatter() {
-    DateFormat dtf = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss zzz");
+    DateFormat dtf = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss zzz", Locale.US);
```

Comment:
Ok got it.
It seems to me the important thing here is the language, not the country, so maybe Locale.ENGLISH is the best thing to use.
As a Brit it always seems strange to me when people chose USA as the canonical example of English !